tests/integration/parsers/chemical_file_formats/test_molecule.py

```diff
@@ -1,21 +1,21 @@
 import os
 
 from tests.fixtures.structural.references import INCHI_DATA
-from tests.integration import IntegrationTestBase
 from express.parsers.molecule import MoleculeParser
+from tests.integration.parsers.chemical_file_formats import ChemicalFileFormatTestBase
```

Comment:
Maybe `structural_data` and `StructuralDataTestBase`?

Reply:
Since there is no use case for a base test class here for now, structural data tests are just inheriting from `TestBase` as in the mention below.